.install/macos.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+brew install make cmake
```

Comment:
You're not getting a guaranteed version of cmake here. Why not use the same mechansim as Linux?